src/resources/projects/website/listing/_pagination.ejs.md

```diff
@@ -1,8 +1,8 @@
-<div class="listing-no-matching d-none"><%- listing.utilities.localizedString("listing-page-no-matches")%></div>
-<% if (listing["page-size"] < items.length) { %>
 ```{=html}
+<div class="listing-no-matching d-none"><%= listing.utilities.localizedString("listing-page-no-matches") %></div>
```

Comment:
Same here: `localizedString()` is for user input value, and we don't know for sure that no special char won't be used. Isn't it safer to keep `<%-` ? 🤔 
is this to support HTML content for `listing-page-no-matches` ?